### ISSUE TITLE
import type * from `rxjs` error

### ISSUE THREAD

**Issue by Crocsx** [diegodoumecq/joymap#9]
Hello,

I recently discovered this package which looks really cool, but I get a problem when using it in my angular app:

I did `npm install joymap` but at compilation time I get the following

```
ERROR in ../node_modules/joymap/bin/src/streamModule/stream.d.ts:1:13 - error TS1005: '=' expected.

1 import type * as rxjs from 'rxjs';
              ~
../node_modules/joymap/bin/src/streamModule/stream.d.ts:1:18 - error TS1005: ';' expected.

1 import type * as rxjs from 'rxjs';
                   ~~~~
../node_modules/joymap/bin/src/streamModule/stream.d.ts:1:23 - error TS1005: ';' expected.

1 import type * as rxjs from 'rxjs';
                        ~~~~
../node_modules/joymap/bin/src/streamModule/stream.d.ts:1:28 - error TS1005: ';' expected.

1 import type * as rxjs from 'rxjs';
                             ~~~~~~
../node_modules/joymap/bin/src/streamModule/stream.d.ts:2:13 - error TS1005: '=' expected.

2 import type * as operators from 'rxjs/operators';
              ~
../node_modules/joymap/bin/src/streamModule/stream.d.ts:2:18 - error TS1005: ';' expected.

2 import type * as operators from 'rxjs/operators';
                   ~~~~~~~~~
../node_modules/joymap/bin/src/streamModule/stream.d.ts:2:28 - error TS1005: ';' expected.

2 import type * as operators from 'rxjs/operators';
                             ~~~~
../node_modules/joymap/bin/src/streamModule/stream.d.ts:2:33 - error TS1005: ';' expected.

2 import type * as operators from 'rxjs/operators';
                                  ~~~~~~~~~~~~~~~~
 
```

I remove the `import type` to `import` and it works ok.

**Comment by diegodoumecq** [comment]
Thanks for the compliment! I've just published version 2.2.2 which includes a fix for this issue.

**Comment by Crocsx** [comment]
@diegodoumecq thank you, It works great after fixing the few changes you made.

Just a last thing, I think your forgot a log in the app cause I get 

http://localhost:4200/#/flight

**Comment by diegodoumecq** [comment]
Yes indeed! Sorry about that, seems like I've been getting careless. Now fixed in version 2.2.4

Oh, also, I've changed how the library is exported so do let me know if I broke something for you (so far I think everything is working correctly)

**Comment by Crocsx** [comment]
Thank you !

I just use it for basic things at the moment (axis event, and button event). After some minor changes to how event works, my app works great like before.

I keep you updated if I found any problem.

Thank you for your work
